packages/react-dev-utils/ModuleScopePlugin.js

```diff
@@ -18,10 +18,11 @@ class ModuleScopePlugin {
 
   apply(resolver) {
     const { appSrc } = this;
-    resolver.plugin('file', (request, callback) => {
+    resolver.hooks.file.tap('ModuleScopePlugin', (request, callback) => {
```

Comment:
second argument is actually not a callback anymore. Not sure how to deal with it due to lack of documentation.

Reply:
For now you can reference the documentation of [tapable](https://github.com/webpack/tapable#readme). I believe you want to use `tapAsync` in this case.

Reply:
Thanks @simonkberg!   While thinking about it, since the plugin isnt doing any async should be valid to use .tap,  return early where possible then just throw the error instead of passing it to the callback? What do you think?